convex/game/agents.ts

```diff
@@ -360,6 +325,7 @@ export const agentRememberConversation = internalAction({
     uuid: v.string(),
   },
   handler: async (ctx, args) => {
+    // TODO: decide whether to do an activity or walk
```

Comment:
is this still current? this would be handled by `agentDoSomething`, yeah?

Reply:
Good point - yes now it starts walking / doing something before doing any remembering.